CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## Unreleased
+
+### Changed
+
+- The `Http\HttplugBundle\Collector\Collector::getClientStack` method returns only stacks without parent.
+
+### Deprecated
+
+- The `Http\HttplugBundle\Collector\Collector` `getClientStacks` method is deprecated. Use `getClientRootStacks` method instead.
```

Comment:
Should we really deprecate the method AND change its behaviour? If fine with deprecate it and leave its current behaviour.